src/framework/lib/genesisframework/utils.rb

```diff
@@ -44,9 +44,12 @@ def self.facter
         end
       end
 
-      def self.log subsystem, message
-        logline = subsystem.to_s + " :: " + message
-        puts logline
+      def self.log subsystem, message, level = nil
```

Comment:
I was thinking using the Collins logging api values, which uses `Collins::API::Logging::log_level_from_string(level)` (private, but useful data there on how to normalize human input like log levels as strings or symbols). https://github.com/tumblr/collins/blob/master/support/ruby/collins-client/lib/collins/api/logging.rb#L155